apps/api/src/app/inbox/usecases/update-preferences/update-preferences.usecase.ts

```diff
@@ -176,7 +176,7 @@ export class UpdatePreferences {
       channels: preference.channels,
       organizationId: command.organizationId,
       environmentId: command.environmentId,
-      subscriberId: command.subscriberId,
+      subscriberId: subscriber._id,
```

Comment:
I have made a similar change in my [PR here](https://github.com/novuhq/novu/pull/6475)
Can you please take a look at that